<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string>Position</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>81</width>
          <height>46</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="xPos">
           <property name="text">
            <string>X : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="yPos">
           <property name="text">
            <string>Y : </string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="imageLayout">
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="cursor" stdset="0">
         <cursorShape>CrossCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionOpen_Image"/>
    <addaction name="actionAdd_Image_To_Stack"/>
    <addaction name="actionSave_Image"/>
   </widget>
   <widget class="QMenu" name="menuCompression">
    <property name="title">
     <string>C&amp;ompressing</string>
    </property>
    <addaction name="actionMSE"/>
    <addaction name="actionGenerate_Rainbow"/>
    <addaction name="actionFnS"/>
    <addaction name="actionDCT"/>
    <addaction name="actionMPEG"/>
   </widget>
   <widget class="QMenu" name="menuAnalyse">
    <property name="title">
     <string>Anal&amp;yse</string>
    </property>
    <widget class="QMenu" name="menuFilter">
     <property name="title">
      <string>&amp;Filter</string>
     </property>
     <addaction name="actionGaussian"/>
     <addaction name="actionSigma"/>
     <addaction name="actionMedian"/>
     <addaction name="actionAverage"/>
    </widget>
    <addaction name="menuFilter"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCompression"/>
   <addaction name="menuAnalyse"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Image">
   <property name="text">
    <string>&amp;Open Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGaussian">
   <property name="text">
    <string>&amp;Gaussian</string>
   </property>
  </action>
  <action name="actionSigma">
   <property name="text">
    <string>&amp;Sigma</string>
   </property>
  </action>
  <action name="actionMedian">
   <property name="text">
    <string>&amp;Median</string>
   </property>
  </action>
  <action name="actionAverage">
   <property name="text">
    <string>&amp;Average</string>
   </property>
  </action>
  <action name="actionMSE">
   <property name="text">
    <string>&amp;MSE</string>
   </property>
  </action>
  <action name="actionGenerate_Rainbow">
   <property name="text">
    <string>&amp;Generate Rainbow</string>
   </property>
  </action>
  <action name="actionFnS">
   <property name="text">
    <string>&amp;Floyd&amp;&amp;Steinberg</string>
   </property>
  </action>
  <action name="actionFloyd_Steinberg">
   <property name="text">
    <string>Floyd&amp;Steinberg</string>
   </property>
  </action>
  <action name="actionQuantify">
   <property name="text">
    <string>&amp;Quantify</string>
   </property>
  </action>
  <action name="actionSave_Image">
   <property name="text">
    <string>&amp;Save Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDCT">
   <property name="text">
    <string>&amp;Jpeg Compression (View)</string>
   </property>
  </action>
  <action name="actionOpen_Second_Image">
   <property name="text">
    <string>Add Image to Stack</string>
   </property>
  </action>
  <action name="actionMPEG">
   <property name="text">
    <string>M&amp;PEG</string>
   </property>
  </action>
  <action name="actionAdd_Image_To_Stack">
   <property name="text">
    <string>Add Image To Stack</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
